core/src/apps/ethereum/layout.py

```diff
@@ -97,7 +97,7 @@ def require_confirm_unknown_token(
     ctx: Context, address_bytes: bytes
 ) -> Awaitable[None]:
     contract_address_hex = "0x" + hexlify(address_bytes).decode()
-    return confirm_address(
+    return confirm_address(  # type: ignore [Expression of type "Coroutine[Any, Any, Awaitable[None]]" cannot be assigned to return type "Awaitable[None]"]
```

Comment:
Type:ignore removed in [39b5e05](https://github.com/trezor/trezor-firmware/pull/2252/commits/39b5e054aedf1f6a28946ce8b74427a1692e4d93)

It is a little interesting for me that type-checkers are taking the `tt_v2` layouts for their analysis.

In my editor, even when I `Go to definition`, it goes to `tt_v2`, which is annoying, because things there are usually not implemented